main.sh

```diff
@@ -33,6 +35,7 @@ post () {
   fi
 
   # POST to federalist's build finished endpoint && POST to federalist-builder's build finished endpoint
+  set +o pipefail
```

Comment:
Yes, this is in the `post()` function which isn't called until the script exists. So unless I'm mistaken, at this point the `/app/*.sh` scripts should have all executed.